### PR TITLE
Update to 3.30.1

### PR DESCRIPTION
**Pull request by uajain**

### CODE REVIEW

**Comment by uajain** [comment]
There's a small problem:

Even though there is no changes for the `liboauth` module, I encounter a build failure there as : https://paste.gnome.org/pehehvrqg

I don't get this build failure on master's manifest.

Looking closely, there is an addition `-fstack-protector-strong` flag coming in. I don't see this flag configured while building master's manifest. I don't know from where it's coming from.

@debarshiray Can you take a quick look?

**Comment by phako** [comment]
TL;DR: liboauth does not compile against OpenSSL 1.1, switch to NSS. 

See your own history: https://gitlab.gnome.org/GNOME/gnome-photos/commit/886d93bf2542fcddf7b7cae56dee20254e26b945

File: org.gnome.Photos.json [hunk 1]
```diff
@@ -84,8 +84,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/libdazzle/3.28/libdazzle-3.28.4.tar.xz",
-                    "sha256": "ebdc1b5e57454310ada72ee81495b31bae468f8246728c8cecae12577f60bd27"
+                    "url": "https://download.gnome.org/sources/libdazzle/3.30/libdazzle-3.30.0.tar.xz",
```
Comment (debarshiray) [inline org.gnome.Photos.json]: Umm... why not go all the way to `3.30.1`?

File: org.gnome.Photos.json [hunk 2]
```diff
@@ -292,8 +299,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://download.gnome.org/sources/gnome-photos/3.28/gnome-photos-3.28.1.tar.xz",
-                    "sha256": "ff8684c3d6f64301ea49271a80c01322b3bd8eed8ce8f4779c4880280cfdd1e0"
+                    "url": "https://download.gnome.org/sources/gnome-photos/3.30/gnome-photos-3.30.0.tar.xz",
```
Comment (debarshiray) [inline org.gnome.Photos.json]: Ditto about `3.30.1`.

**Review by debarshiray — CHANGES_REQUESTED** [review]
By the way, we should also update LibRaw to `0.19.0`.

**Comment by debarshiray** [comment]
I already merged the bit about switching `grilo` to Meson.

Thanks for working on this!

**Comment by uajain** [comment]
Libraw has a patch in the tree. That's why I didn't touch the modules which requires a patch,

**Comment by phako** [comment]
The patch applies cleanly to 0.19.0, FWIW

**Comment by debarshiray** [comment]
Ok, pushed the `LibRaw` update as commit 915519171e2874e6.

**Comment by nedrichards** [comment]
bot, build

**Comment by debarshiray** [comment]
> bot, build

Success: https://flathub.org/builds/#/builders/2/builds/5129 !

**Comment by debarshiray** [comment]
I stuck the URL of this PR in the commit message and merged. Thanks, everybody!

**Pull request closed without merging**